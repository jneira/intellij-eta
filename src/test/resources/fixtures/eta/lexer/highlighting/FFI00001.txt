ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('FFI00001')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITforeign ('foreign')
WHITE_SPACE (' ')
ITimport ('import')
WHITE_SPACE (' ')
ITvarid ('ccall')
WHITE_SPACE (' ')
ITvarid ('unsafe')
WHITE_SPACE (' ')
ITstring ('"')
ITstring ('wool-common.h wool_init')
ITstring ('"')
WHITE_SPACE (' ')
ITvarid ('initWool')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITconid ('CInt')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('Ptr')
WHITE_SPACE (' ')
ITconid ('CString')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('IO')
WHITE_SPACE (' ')
ITconid ('CInt')
WHITE_SPACE ('\n\n')
ITforeign ('foreign')
WHITE_SPACE (' ')
ITexport ('export')
WHITE_SPACE (' ')
ITvarid ('ccall')
WHITE_SPACE (' ')
ITvarid ('id')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITconid ('Int')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('Int')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('IO')
WHITE_SPACE (' ')
ITconid ('CString')
