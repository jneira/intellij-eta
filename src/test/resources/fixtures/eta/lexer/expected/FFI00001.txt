ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('FFI00001')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITvocurly ('')
ITvarid ('foreign')
WHITE_SPACE (' ')
ITimport ('import')
WHITE_SPACE (' ')
ITvarid ('ccall')
WHITE_SPACE (' ')
ITvarid ('unsafe')
WHITE_SPACE (' ')
ITstring ('"wool-common.h wool_init"')
WHITE_SPACE (' ')
ITvarid ('initWool')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITconid ('CInt')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('Ptr')
WHITE_SPACE (' ')
ITconid ('CString')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('IO')
WHITE_SPACE (' ')
ITconid ('CInt')
WHITE_SPACE ('\n\n')
ITsemi ('')
ITvarid ('foreign')
WHITE_SPACE (' ')
ITvarid ('export')
WHITE_SPACE (' ')
ITvarid ('ccall')
WHITE_SPACE (' ')
ITvarid ('id')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITconid ('Int')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('Int')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
ITconid ('IO')
WHITE_SPACE (' ')
ITconid ('CString')